docs/concepts/personal-access-token.mdx

```diff
@@ -1,36 +1,24 @@
 ---
 id: personal-access-token
-title: Personal access tokens
+title: Authorization and API Keys
```

Comment:
```suggestion
title: Authorization and API keys
```

Reply:
I've looked at other documentation sets (including Google and Stripe) and `API keys` (`keys` non-capitalized) is the commonly used approach. Please apply that across this PR.